gulpfile.js

```diff
@@ -146,8 +146,15 @@ function createWebpackConfig(defines, output) {
   var skipBabel = bundleDefines.SKIP_BABEL ||
                   process.env['SKIP_BABEL'] === 'true';
 
+  // Required to expose e.g., the `window` object.
+  output.globalObject = 'this';
+
   return {
+    mode: 'production',
```

Comment:
As can be seen in https://webpack.js.org/concepts/mode/#usage, this mode apparently enables a number of optimizations. Note in particular `UglifyJsPlugin`, which implies that code will now be minified by default!

Since there's a separate `gulp minified` command already, with necessary options being passed, it seems that you want to *explicitly* opt-out of this new Webpack default by specifying `optimization: { minimize: false, },` below (and probably in the examples/webpack/webpack.config.js file too).

Reply:
Done. I noticed that `gulp dist-install` already minifies the bundle, so you're right about that this can also be removed in the example (not sure why it was there before).